zaza/openstack/charm_tests/tempest/templates/tempest_v2.j2

```diff
@@ -42,23 +43,28 @@
 [identity-feature-enabled]
 api_v2 = true
 api_v3 = false
+{% endif %}
 
 [image]
```

Comment:
maybe could take a glance enabled service check here but probably no big deal if glance is disabled anyway